src/client/app/redux/actions/map.ts

```diff
@@ -2,24 +2,20 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
```

Comment:
For the record, this file needs changes to not use State directly, change to lates Redux usage, and maybe more. This is likely a future work.

Reply:
Noted for future work